### PR TITLE
add PyCall Compat

### LINKED ISSUE #4: add Compat PyCall

### PR DESCRIPTION
**Pull request by AtsushiSakai**
fix #4 
Ref: https://github.com/JuliaRegistries/General/pull/15745

### CODE REVIEW

**Comment by giordano** [comment]
Note that you need to trigger registration again as you did before, without changing the version number

**Comment by AtsushiSakai** [comment]
@JuliaRegistrator register

**Comment by giordano** [comment]
I don't know if that works in pull requests, I always comment on the commit page

**Review by AtsushiSakai — COMMENTED** [review]
@JuliaRegistrator register

File: Project.toml [hunk 1]
```diff
@@ -9,6 +9,7 @@ PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
 
 [compat]
 julia = "1.4"
```
Comment (AtsushiSakai) [inline Project.toml:11]: @JuliaRegistrator register

**Comment by giordano** [comment]
No, this commit page: https://github.com/AtsushiSakai/SciPy.jl/commit/a110be25a15900240db2fc31a4fa27e380fc0453

**Comment by AtsushiSakai** [comment]
@giordano Thank you!! The CI passed.

**Comment by giordano** [comment]
Awesome!

**Pull request merged**